### ISSUE TITLE
[Feature Request] An option to open the feeds page as a popup / panel attached to the toolbar icon instead of in a new tab

### ISSUE THREAD

**Issue by FatOrangutan** [ancestorak/brief#328]
It would be great if Brief can show the feeds without leaving the current tab. [Bamboo RSS](https://addons.mozilla.org/en-US/firefox/addon/bamboo-feed-reader/) had a similar feature.

**Comment by tanriol** [comment]
How do you expect that panel to be used? Just a list of feeds with new item counts?

**Comment by FatOrangutan** [comment]
No, basically just use the main /ui/brief.xhtml in the popup itself. So have exactly the same features, same look etc., except instead of opening a new tab every time, it shows up in the toolbar icon's panel, without leaving the current tab.
